pepper-apis/src/main/java/org/broadinstitute/ddp/DataDonationPlatform.java

```diff
@@ -617,14 +620,4 @@ private static void initSqlCommands(Config sqlConfig) {
         DBUtils.loadDaoSqlCommands(sqlConfig);
     }
 
-    /**
-     * Allow to specify the exclusion of a path from the execution of a filter
-     *
-     * @param filterPath     the path for which the filter is applicable
-     * @param pathsToExclude the paths to exclude the execution of the filter
-     * @param filter         the filter
-     */
-    public static void beforeWithExclusion(String filterPath, String[] pathsToExclude, Filter filter) {
-        before(filterPath, new ExcludePathFilterWrapper(filter, Arrays.asList(pathsToExclude)));
-    }
```

Comment:
Since I will be creating a new home for custom filters (FilterUtils), I am going to move this custom filter there too.